backend-shared/plannershared.cpp

```diff
@@ -63,20 +63,42 @@ void plannerShared::set_min_switch_duration(int value)
 
 double plannerShared::bottomsac()
 {
-	return qPrefDivePlanner::bottomsac() / 1000.0;
+	return (qPrefUnits::volume() == units::LITER) ?
+				qPrefDivePlanner::bottomsac() / 1000.0 :
+				ml_to_cuft(qPrefDivePlanner::bottomsac()
+#ifdef SUBSURFACE_MOBILE
+					* 100 // cuft without decimals (0 - 300)
+#endif
```

Comment:
so we are returning an un-typed floating point number, with different scaling depending on the UI.

I know we talked about this before - the idea that we prepare UI text in shared code.

Hmm - trying to figure out where to take this comment.

I guess all I want to express is that I struggle with this. It feels wrong. But I don't know if I should push for something else. And what exactly to push for.